miso-web/src/main/webapp/scripts/form_run.js

```diff
@@ -133,6 +133,18 @@ FormTarget.run = (function($) {
           getItemValue: function(item) {
             return item.value;
           }
+        }, {
+          title: 'Index Sequencing',
+          data: 'dataManglingPolicy',
+          type: 'dropdown',
+          source: Constants.dataManglingPolicies,
```

Comment:
It might make sense to be able to set this back to `null` as _Instrument Default_ or something like that.

Reply:
You already can - the field is not marked `required` and `nullLabel` is "Instrument Default"